### PR TITLE
Testing some stuff re: CI

### PR DESCRIPTION
**Pull request by EricBorczuk**

### CODE REVIEW

File: testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java [hunk 1]
```diff
@@ -92,7 +92,9 @@ public void setup(StargateConnectionInfo cluster) throws IOException {
 
   @AfterEach
   public void teardown() {
-    session.execute(String.format("drop keyspace %s", keyspace));
+    if (Math.random() < 0.5) {
+      session.execute(String.format("drop keyspace %s", keyspace));
+    }
```
Comment (dimas-b) [inline testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java:97]: Do we intend to merge this?

Comment (EricBorczuk) [reply testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java]: No - I'm just messing around for now

Comment (dimas-b) [reply testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java]: Ah... mind converting the PR to "Draft"? :-)

Comment (EricBorczuk) [reply testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java]: Will do sorry :P

**Pull request closed without merging**